test

```diff
@@ -4,11 +4,25 @@
 # ./test
 # ./test -v
 #
+#
+# Run specified test pass
+#
+# PASSES=functional ./test -v
+# PASSES=integration ./test -v
+#
+#
 # Run tests for one package
 #
 # PKG=./wal ./test
 # PKG=snap ./test
 #
+#
+# Run specified unit tests in one package
+#
+# PASSES=unit PKG=./wal TESTCASE=TestNew ./test -v
+# PASSES=unit PKG=./wal TESTCASE=\bTestNew\b ./test -v
```

Comment:
Should we explain what the `\b`s mean? It's non-obvious.

Reply:
Also, this only applies to `unit` right? Should it be applied to all PASSES?

Reply:
Remove `-v`?

Reply:
Good points! I will 1. add comment for \b, 2, remove -v, 3. make it available for all PASSES.